core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -32,6 +32,7 @@ import org.apache.spark.executor._
 import org.apache.spark.scheduler._
 import org.apache.spark.storage._
 import org.apache.spark._
+import org.apache.hadoop.hdfs.web.JsonUtil
```

Comment:
Note this breaks Mapr3 builds. `<hadoop.version>1.0.3-mapr-3.0.3</hadoop.version>` 

The version of mapr3 should be upgraded then to `1.0.3-mapr-3.1.1`